apps/member-profile/app/shared/components/company-review.tsx

```diff
@@ -49,26 +57,45 @@ export const CompanyReview = ({
   reviewerLastName,
   reviewerProfilePicture,
   reviewedAt,
+  studentId,
   text,
   title,
+  workExperiencesId,
 }: CompanyReviewProps) => {
   return (
     <Card>
-      <header className="flex items-center gap-1">
-        <CompanyReviewer
-          reviewerFirstName={reviewerFirstName}
-          reviewerLastName={reviewerLastName}
-          reviewerId={reviewerId}
-          reviewerProfilePicture={reviewerProfilePicture}
-        />
-
-        <Text color="gray-500" variant="sm">
-          &bull;
-        </Text>
-
-        <Text color="gray-500" variant="sm">
-          {reviewedAt}
-        </Text>
+      <header className="space-between flex items-center">
```

Comment:
When I was testing locally it seems that the `Edit` button is not getting moved to the right of the box, but rather clumped with the other elements on the left. I believe the issue is here, and should be fixed if instead we just add

        {studentId === reviewerId && (
          <div className="ml-auto">
            <Tooltip>
              <TooltipTrigger aria-label="Edit Review">
                <Link
                  to={generatePath(Route['/profile/work/:id/review/edit'], {
                    id: workExperiencesId!,
                  })}
                >
                  <Edit />
                </Link>
              </TooltipTrigger>
              <TooltipContent>
                <TooltipText>Edit Review</TooltipText>
              </TooltipContent>
            </Tooltip>
          </div>
        )}

The point is we don't have to change the formatting/divs/headers for any of the other elements, but rather only modify code for moving this new element to the right of the box

Reply:
Ok @tomas-salgado, made the changes. Thanks for the feedback!